pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/tasks/SphinxDocumentationTask.groovy

```diff
@@ -39,11 +39,15 @@ class SphinxDocumentationTask extends AbstractPythonMainSourceDefaultTask {
     public DocType type
 
     @Override
-    public void preExecution() {
-        args(pythonDetails.virtualEnvironment.findExecutable('sphinx-build').absolutePath,
-            '-b', type.builderName,
-            project.file(component.docsDir).getAbsolutePath(),
-            "${ getDocDir().getAbsolutePath() }")
+    void preExecution() {
+        if (project.file(component.docsDir).exists()) {
+            args(pythonDetails.virtualEnvironment.findExecutable('sphinx-build').absolutePath,
+                '-b', type.builderName,
+                project.file(component.docsDir).getAbsolutePath(),
+                "${getDocDir().getAbsolutePath()}")
+        } else {
+            project.logger.warn("Sphinx docs dir doesn't exist.  Aborting")
```

Comment:
This isn't really an abort. We should do this as a `Task.onlyIf()`vs having the task execute.